pkg/visor/api.go

```diff
@@ -114,38 +114,41 @@ func (v *Visor) Summary() (*Summary, error) {
 		return nil, fmt.Errorf("failed to get IPs of interface %s: %w", defaultNetworkIfc, err)
 	}
 
-	summary := &Summary{
+	overview := &Overview{
 		PubKey:          v.conf.PK,
 		BuildInfo:       buildinfo.Get(),
 		AppProtoVersion: supportedProtocolVersion,
 		Apps:            v.appL.AppStates(),
-		Transports:      summaries,
+		Transports:      tSummaries,
 		RoutesCount:     v.router.RoutesCount(),
 	}
 
 	if len(localIPs) > 0 {
 		// should be okay to have the first one, in the case of
 		// active network interface, there's usually just a single IP
-		summary.LocalIP = localIPs[0].String()
+		overview.LocalIP = localIPs[0].String()
 	}
 
-	return summary, nil
+	return overview, nil
 }
 
-// ExtraSummary provides an extra summary of a Skywire Visor.
-type ExtraSummary struct {
-	Summary *Summary                        `json:"summary"`
-	Dmsg    []dmsgtracker.DmsgClientSummary `json:"dmsg"`
-	Health  *HealthInfo                     `json:"health"`
-	Uptime  float64                         `json:"uptime"`
-	Routes  []routingRuleResp               `json:"routes"`
+// Summary provides detailed info including overview and health of the visor.
+type Summary struct {
+	Overview     *Overview                      `json:"overview"`
+	Health       *HealthInfo                    `json:"health"`
+	Uptime       float64                        `json:"uptime"`
+	Routes       []routingRuleResp              `json:"routes"`
+	IsHypervisor bool                           `json:"is_hypervisor,omitempty"`
+	DmsgStats    *dmsgtracker.DmsgClientSummary `json:"dmsg_stats"`
+	Port         uint16                         `json:"port"`
```

Comment:
Not sure what we display that for. Is it the `dmsg` port?

Reply:
Yes it's the `dmsg` port.